Sources/TuistCache/ContentHashing/TargetScriptsContentHasher.swift

```diff
@@ -41,6 +41,9 @@ public final class TargetScriptsContentHasher: TargetScriptsContentHashing {
             stringsToHash.append(
                 contentsOf: (script.outputPaths + script.outputFileListPaths).map { $0.relative(to: sourceRootPath).pathString }
             )
+            if let dependencyFile = script.dependencyFile {
+                stringsToHash.append(dependencyFile.relative(to: sourceRootPath).pathString)
```

Comment:
Given this is an input file, this should hash the file, not just its path, as done on line 37
```suggestion
                pathsToHash.append(dependencyFile)
```

Reply:
Then I would do what it's done above for the other files, if the name contains a $ it's dynamic and hence hashed as string, otherwise it's static and it's hashed as file, WDYT?

Reply:
Yeap done.